### DISCUSSION TITLE
Accounting for import and export (Switzerland as example)

### DISCUSSION THREAD

**Discussion by jorimrosenberg** [electricitymaps/electricitymaps-contrib#5289] [Q&A] [answered]
Hey guys,

After regulariy checking out ElectricityMap for the last couple of months, specifically for Switzerland, I've noticed that the values are highly dependent on imports, which makes perfect sense. However, what bothers me is the following: Today on the 13.04.23 at 10:00, Switzerland has imported pretty much the exact amount of electricity from AT, DE and FR that it has exported to IT-CNO (about 4.2 MW). With the way ElectricityMap accounts for imports and exports (as read [here](https://www.electricitymaps.com/blog/flow-tracing)), the imports are added to the local production and CO2 emissions are calculated from this mix for this zone. The exported electricity has then the zone's value as CO2 emission. In the end, as the imported electricity has higher CO2 emissions than the local production in my case, this means that the CO2 emissions of Switzerland are overestimated, and the one for Italy are underestimated.

My argument is that as the electricity that is imported to CH is exported to IT, the higher production of AT, DE and FR is caused by IT, and therefore should be accounted to IT and not CH. The electricity is merely passing through CH, not caused by swiss consumption. If my math is correct, the value for Switzerland should have been at about 100 gCO2/kWh instead of 125 gCO2/kWh stated by ElectricityMap, which is a quite substantial difference.

I'm curious to hear your opinion on this topic.

Cheers
Jorim

**Comment by VIKTORVAV99** [comment]
We have a pretty extensive blog post on this matter and our opinion is very unlikely to change on the matter but please do have a read if you are interested: https://www.electricitymaps.com/blog/flow-tracing

**Reply by jorimrosenberg** [reply]
Thanks for the quick response. That's the article I've read prior to posting this (I have also linked it), but it doesn't say anything about this issue so I was wondering what the reason behind choosing this way of accounting is?

**Reply by VIKTORVAV99** [reply]
Basically the reason for this is that it's impossible to tell where electricity comes from once it's mixed. You don't know if you are getting electricity from AT, DE or from local hydro power, so the only fair to attribute the carbon intensity is to make it the same one for all. And since there is no dedicated powerlines that run from Germany to Italy there is no way to avoid that the German carbon intensity affects the Swiss carbon intensity if that makes sense.

The blog post I linked use the smoothie analogy, i.e. that once you have mixed it it's impossible to separate it back to its original parts, the same way goes for electricity.

**Reply by jorimrosenberg** [reply]
Well, one could first balance the country isolated (e.g. 10GW consumption, 9GW production), then add the 1GW of imported electricity as a mix of all imports, and the rest is then mixed to the countries it is exporting to. Of course in the grid, one cannot distinguish the electrons produced by DE from the ones produced by CH, but this method would account the overproduction of certain countries to the underproduction of others, instead of smothing it out when going through other countries.

So with the smoothie analogy: We have smoothie1 (local production) and smoothie2 (imports). Instead of mixing both smoothies completely, smoothie2 is only partially mixed with smoothie1 and the rest exported to neighbouring countries.

**Reply by VIKTORVAV99** [reply]
I think a better example would be to try and make two different smoothies with just one mixer at the same time, it's simply impossible to separate the two once they have become blended.

Sure we could calculate the import and exports for a country and find the "transit" amount and that would be really easy to do for a zone with just 2 connections. But if a zone has let's say 12 connections like Germany does it would be impossible to attribute the imports from one zone to the export of another. There is no way to say that the export to Sweden is coming only from France and the import from Norway is going only to Denmark for example.

Now if we did it like this we would still have to mix all the imports to make it even possible.

But even so this would not be a fair attribution for any party involved. Electricity by its very nature can't be split up once it's combined and we have established a methodology that is built on this and is trusted and proven.

**Reply by mirkoschaefer** [reply, marked as answer]
Hi @jorimrosenberg , I think what you are describing is the difference between what I call "direct coupling" and "aggregated coupling" for the flow tracing method. We included a discussion about that in a paper, free version here:
https://arxiv.org/abs/2008.00893
Here is a video presentation with a simple example showing the difference:
https://videoportal.uni-freiburg.de/video/Tracing-carbon-dioxide-emissions-in-the-European-electricity-market/8296bd92990ccbd7e176be95dc1b1bc7

Both approaches are valid, but use a slightly different interpretation of the method - personally, I would use direct coupling for a detailed network representation, and aggregated coupling for a more clustered representation.

**Reply by jorimrosenberg** [reply]
@mirkoschaefer Yes that's exactly what I tried to describe, many thanks for pointing it out! And nice work, I would have expected the impact of the choice of coupling method to be much higher for CH than what you show in Fig. 5.

So @VIKTORVAV99 it would be fairly easy to implement the "aggregated coupling" method with the same input data. Basically, you would mix all the imports and just add the portion needed for the specific country and by-pass the rest to all export countries.

I still think it would be more appropiate to do aggregated coupling as it allocates CO2 emission to the polluter more directly. Direct coupling gives a kind of blurred out representation, and countries sort of profit randomly from their neighbours effort to reduce CO2 emissions. But I guess this is more a philosophical question, and I see that you chose direct coupling as aggregated coupling is technically infeasible.

**Reply by mirkoschaefer** [reply]
@jorimrosenberg What you see in Fig. 5 are the average values. We found that for the given data, the impact on the average carbon intensity often is not very high, but for individual hours it actually can be very significant (as you can see in Fig. 4). If I recall correctly, Estland is a good example of the difference between aggregated and direct coupling. It usually has a high share of power generation from coal, but passes through (low-carbon) power flow from Finland and Sweden. In the direct coupling, this transient (low-carbon) flow reduces its consumption-based emission intensity (also visible in Fig. 4 in our paper). Nevertheless, these results should be taken with a bit of care, I also vaguely remember that the data quality in this region is not ideal, leading to necessary corrections to obtain a balanced flow pattern.

**Comment by nielsle** [comment]
If you just look at the web interface, then it is difficult to guess if C02-emissions are assigned to the country of production or to the country where the electricity is being used. 

I think that it would be great to have a clarifying sentence on the website itself.
